src/components/data/listing/Listing.js

```diff
@@ -561,6 +589,53 @@ function Listing(props) {
             >
                 <PublicLinks paths={getSelectedPaths()} />
             </DEDialog>
+            <DEDialog
+                open={confirmDOIRequestDialogOpen}
+                onClose={() => setConfirmDOIRequestDialogOpen(false)}
+                baseId={ids.DOI_CONFIRM}
+                title={t("requestDOI")}
+                actions={
+                    <>
+                        <Button
+                            onClick={() =>
+                                setConfirmDOIRequestDialogOpen(false)
+                            }
+                        >
+                            {t("cancel")}
+                        </Button>
+                        <Button
+                            color="primary"
+                            onClick={() => {
+                                setConfirmDOIRequestDialogOpen(false);
+                                requestDOI({
+                                    folder: selected[0],
+                                    type: "DOI",
+                                });
+                            }}
+                        >
+                            {t("needDoi")}
+                        </Button>
+                    </>
+                }
+            >
+                <Trans
+                    t={t}
+                    i18nKey="requestDOIPrompt"
+                    components={{
+                        manual: <ExternalLink href={constants.DOI_GUIDE} />,
+                    }}
+                />{" "}
```

Comment:
Would removing the `{" "}` and replacing it with `</br>` put the next message on a new line?  
Since these two messages change perspectives (first message is in the second person with using "you" and the second message is in the first person with using "I"), it'd be nice if these two messages didn't look like they were part of the same paragraph.